### PR TITLE
Housekeeping Changes

### PR DESCRIPTION
**Pull request by xueharry**
Hey @arthurian, can you take a look at the housekeeping changes? Thanks.

### CODE REVIEW

File: canvas_syllabus_export/settings/aws.py [hunk 1]
```diff
@@ -16,6 +16,8 @@
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
+# Base URL for Canvas instance
+BASE_URL = "https://canvas.harvard.edu/api"
```
Comment (arthurian) [inline canvas_syllabus_export/settings/aws.py]: Can you make this a secure setting as well?

File: canvas_syllabus_export/settings/local.py [hunk 2]
```diff
@@ -16,6 +16,9 @@
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
+# Base URL for Canvas instance
+BASE_URL = "https://canvas.harvard.edu/api"
```
Comment (arthurian) [inline canvas_syllabus_export/settings/local.py]: Note: this is fine for local development (hard-coded). So you can leave this as-is.

**Comment by arthurian** [comment]
@xueharry It looks like there are some conflicts with the merge. Let me know if you need help resolving the conflicts. I'm guessing the conflict is because of the change I made to ```requirements.txt```. You'll have to merge *Harvard-ATG:master* into *xueharry:master* first. Resolve the conflicts there, and then it should be able to merge cleanly back into *Harvard-ATG:master*.

**Comment by xueharry** [comment]
@arthurian Merge conflict fixed

**Comment by arthurian** [comment]
:+1:Looks good. Feel free to merge the PR.

File: canvas_syllabus_export/settings/aws.py [hunk 3]
```diff
@@ -110,9 +115,13 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
-    }
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': SECURE_SETTINGS.get('db_default_name', 'canvas_syllabus_export'),
+        'USER': SECURE_SETTINGS.get('db_default_user', 'postgres'),
```
Comment (bermudezjd) [inline canvas_syllabus_export/settings/aws.py]: This should also default to the project name (canvas_syllabus_export)

**Pull request merged**